envoy_pkg/getenvoy.WORKSPACE

```diff
@@ -68,10 +68,10 @@ rbe_autoconfig(
     create_java_configs = False,
     env = rbe_envs(),
     registry = "gcr.io",
-    repository = "getenvoy-package/rbe-linux-glibc-fips",
+    repository = "master-coder-313422/rbe-linux-glibc-fips",
```

Comment:
You should also be able to configure this as your local repo instead of going to a Google image repository